### PR TITLE
Extract internal API to a submodule (#7)

### PR DESCRIPTION
**Pull request by aerosol**
@zoldar review and/or merge please :)

### CODE REVIEW

File: lib/renderer/renderer.ex [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  github_md: [
+    heading:        " | ",
+    heading_border: " | ",
+    row:            " | ",
+    spacer:         "-"
+  ]
+
+  @default_sheet :org_mode
+
+  def run(rows, [_|_]=cols, opts) do
```
Comment (zoldar) [inline lib/renderer/renderer.ex:31]: `run` implies doing something with/for side effects. `render`, although redundant, would fit here better imho - it also conveys some meaningful action.

File: mix.exs [hunk 2]
```diff
@@ -30,8 +30,9 @@ defmodule Tabula.Mixfile do
   #
   # Type `mix help deps` for more examples and options
   defp deps do
-    [{:earmark, "~> 0.1", only: :dev},
-      {:ex_doc, "~> 0.10", only: :dev}]
+    []
+    #[{:earmark, "~> 0.1", only: :dev},
+      #{:ex_doc, "~> 0.10", only: :dev}]
```
Comment (zoldar) [inline mix.exs:35]: Are these needed or not? It's under version control ffs! ;)

Comment (aerosol) [inline mix.exs:35]: Dang! :sweat:

File: lib/renderer/renderer.ex (hunk 1 above)
Comment (aerosol) [inline lib/renderer/renderer.ex:31]: Agreed

**Comment by zoldar** [comment]
Closing for now. Will probably need a fresh start anyway.

**Pull request closed without merging**